packages/ui/src/ui/pages/tablet_cell_bundles/bundles/BundleEditorDialog/BundleEditorDialog.tsx

```diff
@@ -318,6 +333,15 @@ export function BundleEditorDialog() {
                     },
                 },
                 validator: simpleBundleValidate,
+                tooltip: (
```

Comment:
If I'm not mistaken, now we will never see these fields when BC is off?
Because editing is not permitted for BC=off ((

Reply:
`renderSimpleResourceTab` is used only when BC == 'off' and current user is allowed to write //sys/tablet_cell_bundles/{bundleName} node. So to see the message a user should have write access to a bundle.